lib/ex_doc/formatter/epub/templates/title_template.eex

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"
+      xmlns:epub="http://www.idpf.org/2007/ops">
+  <head>
+    <meta charset="utf-8" />
+    <title><%= config.project %> v<%= config.version %></title>
+    <meta name="generator" content="ExDoc" />
+    <link type="text/css" rel="stylesheet" href="css/stylesheet.css" />
+  </head>
+  <body>
+    <h1><%= config.project %></h1>
+    <h2>v<%= config.version %></h2>
+    <%= if config.logo do %>
+      <%= if Path.extname(config.logo) == ".png" do %>
+        <div><img src="assets/logo.png" alt="Logo"/></div>
+      <% end %>
+      <%= if Path.extname(config.logo) == ".jpg" do %>
+        <div><img src="assets/logo.jpg" alt="Logo"/></div>
+      <% end %>
```

Comment:
@dukex `ExDoc` right now only support `.jpg` and `.png` (see: `ExDoc.Formatter.HTML.process_logo_metadata/1`)

@eksperimental We don't apply any transformation to the image format, we just the rename the file name to `logo`. So, in your alternative you're missing the `.jpg` extension with this:

```html
<div><img src="assets/logo.png" alt="Logo"/></div>
```